### PR TITLE
Swat/defect/fix daemon routers

### PR DESCRIPTION
**Pull request by sshankar19**

### CODE REVIEW

File: lib/setupCustomRoutes.js [hunk 1]
```diff
@@ -13,17 +14,21 @@ function setupCustomRoutes(appDirectory, mainAppName, router) {
         const exportedRouterFunctions = Object.keys(routers);
 
         if(exportedRouterFunctions.length === 0) {
-            console.log('a custom router file was defined, but no exported ' +
+            logMessage('A custom router file was defined, but no exported ' +
```
Comment (hackerrdave) [inline lib/setupCustomRoutes.js]: spelling: "but not exported"

File: lib/setupCustomRoutes.js [hunk 2]
```diff
@@ -13,17 +14,21 @@ function setupCustomRoutes(appDirectory, mainAppName, router) {
...
                 'functions were found. Using default amorphic routes');
             return;
         }
 
         // iterate through all exported router functions and execute.
         for (const routerFunction of exportedRouterFunctions) {
+
             if (typeof routers[routerFunction] === 'function') {
-                return routers.routerSetup(router);
+                logMessage('Evaluating router: ' + routerFunction);
```
Comment (hackerrdave) [inline lib/setupCustomRoutes.js]: should this be debug-level? seems verbose to log every router function

File: lib/setupCustomRoutes.js (hunk 1 above)
Comment (hackerrdave) [reply lib/setupCustomRoutes.js]: this should be "WARN" log right?

File: lib/setupCustomRoutes.js [hunk 3]
```diff
@@ -1,9 +1,10 @@
 'use strict';
 
 const fs = require('fs');
+const logMessage = require('./utils/logger').logMessage;
```
Comment (hackerrdave) [inline lib/setupCustomRoutes.js]: should probably use a logger instance, this is just a console.log

**Review by hackerrdave — DISMISSED** [review]

File: lib/setupCustomRoutes.js (hunk 1 above)
Comment (sshankar19) [reply lib/setupCustomRoutes.js]: I changed this to a console.warn. logMessage at this moment only logs at 1 level. And the spelling makes sense after the concatenation

File: lib/setupCustomRoutes.js (hunk 3 above)
Comment (sshankar19) [reply lib/setupCustomRoutes.js]: We have no logger instance at this point, I can make a ticket for this. But that's not in the scope of this MR

File: lib/setupCustomRoutes.js (hunk 2 above)
Comment (sshankar19) [reply lib/setupCustomRoutes.js]: Yep changed this to debug level

File: lib/setupCustomRoutes.js (hunk 1 above)
Comment (nsaenz7) [reply lib/setupCustomRoutes.js]: for spelling, it's part of longer string continued on next line, `but no exported (next line) functions were found`. this was split so it wasn't a super long line. as far as the warn level, think we want to use the `log` function here and pass in a log level, it looks like logMessage should really be a private helper function of `log` and not used directly.

**Review by srksag-zz — APPROVED** [review]

**Review by nsaenz7 — APPROVED** [review]

**Pull request merged**